rosetta-lang/src/main/java/com/regnosys/rosetta/types/RosettaTypeProvider.xtend

```diff
@@ -186,6 +191,9 @@ class RosettaTypeProvider extends RosettaExpressionSwitch<RType, Map<EObject, RT
 		])
 	}
 	private def RType getRTypeFromCache(String cacheKey, EObject object, Provider<RType> typeProvider) {
+		if (!EXPRESSION_RTYPE_CACHE_KEY_ENABLED) {
+			typeProvider.get()
```

Comment:
Shouldn't there be a `return` here?

Reply:
Actually - you are right. I take it back. Adding the return!